lib/Dialect/Torch/IR/TorchOps.cpp

```diff
@@ -735,6 +735,19 @@ OpFoldResult AtenDimOp::fold(ArrayRef<Attribute> operands) {
   return nullptr;
 }
 
+//===----------------------------------------------------------------------===//
+// AtenIsFloatingPointOp
+//===----------------------------------------------------------------------===//
+
+OpFoldResult AtenIsFloatingPointOp::fold(ArrayRef<Attribute> operands) {
+  if (auto tensorType = getOperand().getType().dyn_cast<BaseTensorType>()) {
+    if (typeid(tensorType).name() == "f")
```

Comment:
You can check if a type is float by doing something like

https://github.com/llvm/torch-mlir/blob/e1b38e74dd0072f9d6d48f7c90e21912fe3e85cd/lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp#L298-L299